storage/remote/store.go

```diff
@@ -176,11 +224,6 @@ func (s *Store) LedgerByHeight(
 	ctx context.Context,
 	blockHeight uint64,
 ) (snapshot.StorageSnapshot, error) {
-	err := s.SetBlockHeight(blockHeight)
```

Comment:
Setting this here could have confusing results, since it may reset the latest height to an earlier block if `ExecuteScript` is called with a past height.

It's already called when committing a block, so we can skip it
https://github.com/onflow/flow-emulator/blob/0273f8fe3807746ac065b64dce30c8deafd7c216/storage/store.go#L213-L216

Reply:
@sideninja keep me honest

Reply:
Yes, looking at this it doesn't need to be here, not sure what was my thinking at the time.